packages/lib/gpc/src/gpcUtil.ts

```diff
@@ -22,6 +21,8 @@ import {
   GPCProofObjectConfig,
   GPCProofTupleConfig,
   PODEntryIdentifier,
+  PODVirtualEntryName,
+  POD_VIRTUAL_ENTRY_IDENTIFIER_REGEX,
```

Comment:
GitHub won't let me comment too far from changes, but something else in this file which needs changing:
- canonicalizeEntryConfig (or the canonicalization process in general) needs to be updated to deal with the different default value for isRevealed on signerPublicKey

Reply:
Does this mean the signer's public key's config should *not* be optional?

Reply:
No, I think it can still be optional, just with a different default.  The goal of canonicalization is that equivalent configs always result in the same output, including deleting unnecessary fields which are set to the default.

So in this case it would delete isRevealed=true.  Actually probably it should be able to delete the whole signerPublicKey config object if it doesn't contain any non-default settings.  That wasn't a case which came up before, since for entries, the presence of an empty entry config is meaningful.  It proves the entry is present in the POD, but you don't need any config to prove the signer's public key is "present" since it's a required thing.

FWIW I don't love the idea of boolean fields which don't default to false, since it's least confusion if absent->false is consistent everywhere.  We could avoid that by changing the config field to isHidden.  However I don't think we should, since I think the benefit of having the signerPublicKey config use the same fields as an entry config is an advantage to clarity which is more than the disadvantage of having a boolean default to true.  Feedback on this tradeoff is welcome, though.